src/generator.js

```diff
@@ -0,0 +1,29 @@
+import faker from 'json-schema-faker';
+import annotations from './annotations';
+
+faker.option({
+  useDefaultValue: true,
+});
+
+export function bodyFromSchema(schema, payload, parser) {
+  const {Asset} = parser.minim.elements;
+  let asset = null;
+
+  try {
+    const body = schema.example || JSON.stringify(faker(schema), null, 2);
+
+    asset = new Asset(body);
+
+    asset.classes.push('messageBody');
+    asset.contentType = 'application/json';
+
+    payload.content.push(asset);
+  } catch (exception) {
+    parser.createAnnotation(annotations.DATA_LOST, parser.path,
+      'Unable to generate fake body out of schema');
```

Comment:
I think this message could be a little more user-friendly. Not sure that everyone will understand what `fake body` means. Perhaps this could read:

> Unable to generate a JSON example message body from JSON Schema.